packages/oraidex-common/src/pairs.ts

```diff
@@ -169,3 +169,23 @@ export const PAIRS_CHART = PAIRS.map((pair) => {
     info: `${assets[0]}-${assets[1]}`
   };
 });
+
+export enum PairAddress {
```

Comment:
i think we can use Record with key get from list Pairs that was configured already so it can avoid missed pair when list Pairs add new